src/main/java/uk/gov/companieshouse/web/accounts/CompanyAccountsWebApplication.java

```diff
@@ -25,6 +32,15 @@ public CompanyAccountsWebApplication(UserDetailsInterceptor userDetailsIntercept
     }
 
     public static void main(String[] args) {
+
+        try {
+            ClassPathScanningCandidateComponentProvider provider = new ClassPathScanningCandidateComponentProvider(true);
+            ValidationHelper.scanPackageForValidationMappings(provider, APPLICATION_MODEL_PACKAGE);
```

Comment:
This is a Spring component and shouldnt need and static methods to make it to work, the act of spring injecting the bean should be enough for it to do everything thats required. perhaps the static methods should be instance methods instead and the scan scanPackageForValidationMappings be called witihn the constructor?